.version

```diff
@@ -1 +1 @@
-21.11
\ No newline at end of file
+21.11
```

Comment:
Please revert this change.

Reply:
Sorry, was trying to make the EditorConfig check green.  Dropped the commit.